src/System.Private.DataContractSerialization/src/System/Runtime/Serialization/SerializableInfo.cs

```diff
@@ -0,0 +1,43 @@
+﻿//-----------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.  All rights reserved.
+//-----------------------------------------------------------------------------
```

Comment:
I think this type should be removed, but if it needs to stay for some reason, the license comment header is incorrect.